src/Hydration.js

```diff
@@ -1,31 +1,33 @@
-class Hydration {
-  constructor(hydrationData) {
-    this.hydrationData = hydrationData;
+import Fitlit from './Fitlit';
```

Comment:
Cool so it looks like we are easily to inherit using the super class to access data smoothly. Looks so much better.